docker/run.sh

```diff
@@ -2,6 +2,9 @@
 
 set -x
 
+AMOC_GRAPHITE_HOST=${AMOC_GRAPHITE_HOST:-127.0.0.1}
```

Comment:
I'm not sure that default endpoint for graphite host should be 127.0.0.1:2003. Is it started in the container?

Reply:
We need to provide any address. Otherwise Amoc inside container would crash. We have fixed `docker/config/sys.config.template`. If we do not provide `AMOC_GRAPHITE_PORT`, it would have invalid syntax. 

I just added `AMOC_GRAPHITE_HOST` to make it more consistent.

When wrong address it provided (like default one), just `exometer` application will crash. Amoc is still operable.